src/components/LikeButton/Likes.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import { updateVotes } from "../../utils";
+import './Likes.css'
+
+
+export default function LikeCount({article}) {
+  const [votes, setVotes] = useState(article.votes) 
+
+
+  function handleClick(event) {
+    event.preventDefault();
+    setVotes((currentVotes) => currentVotes + 1)
+    updateVotes(article.article_id)
+  }
+
+
+  return (
+
+
+      <button className='likes-button' onClick={handleClick}>
+        <span className="heart">&#10084;</span> 
+        <span className="count">{votes}</span>
+      </button>
+
+   
+  );
+}
+
+
```

Comment:
Also while I love the good vibes only feel of being able to upvote, it would be cool if the user could undo their vote
This isn't mandatory though